WcaOnRails/app/views/persons/_results_by_event.html.erb

```diff
@@ -0,0 +1,61 @@
+<div class="results-by-event">
+  <div class="event-selector text-center">
+    <% @results.map(&:event).uniq.each do |event| %>
```

Comment:
I've just realised that the name is misleading. Those are the events that you have RanksSingle or RanksAverage computed for. So magic is not present here! I'll leave this part as it is. Also see [the comment here](https://github.com/thewca/worldcubeassociation.org/pull/1579#discussion_r114319024).